app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -30,9 +30,9 @@ import org.simple.clinic.medicalhistory.MedicalHistoryRepository
 import org.simple.clinic.medicalhistory.OngoingMedicalHistoryEntry
```

Comment:
Oh I get it now. Smart!
Although I am not sure if this an appropriate way to refactor.
Can you merge this commit with the next one though and refactor at once after you change enums to sealed class?

Reply:
I renamed it to what it needs to be called when it becomes a sealed class so that it is extremely easy to refactor. So for example: `SCHEDULED` becomes `Scheduled` across the project and then I convert them to a sealed class.